src/staticanalysis/bot/static_analysis_bot/report/base.py

```diff
@@ -130,8 +130,7 @@ def pluralize(word, nb):
         # Build top comment
         nb = len(issues)
         extras = ''
-        if max_comments is not None and nb > max_comments:
-            extras = ' (only the first {} are reported here)'.format(max_comments)
+        
 
```

Comment:
There are too many blank lines now, you have to remove one.
The build is failing because of a linting error:
```
./static_analysis_bot/report/base.py:133:1: W293 blank line contains whitespace
./static_analysis_bot/report/base.py:135:9: E303 too many blank lines (2)
```

Reply:
i removed the 133rd blank line, and created another pull request, (#1876)

Reply:
You don't need to create a new PR for every requested change, otherwise we'd be flooded with PRs :smile: 
You can just update the existing PR, like you did.

Reply:
Oh lol sorry didn't see that it got updated :p

Reply:
Oh lol sorry didn't see that it got updated :p